screens/EpisodeInputScreen.tsx

```diff
@@ -20,7 +20,10 @@ import { TouchableWithoutFeedback } from "react-native";
 
 let recordingDay: number = 1;
 const convertToMinutes = (time: Date) => {
-  return time.getHours() * 60 + time.getMinutes();
+  return time.toLocaleTimeString("en-US", {
+    hour: "numeric",
+    minute: "2-digit",
+  });
 };
```

Comment:
Just wondering, is this used? It looks like you deleted it.

Reply:
Thanks I never actually deleted this